apify-api/openapi/paths/actors/acts@{actorId}@builds.yaml

```diff
@@ -75,8 +75,8 @@ get:
   x-py-doc-url: https://docs.apify.com/api/client/python/reference/class/BuildCollectionClientAsync#list
 post:
   tags:
-    - Actors/Build collection
-  summary: Build actor
+    - Builds
```

Comment:
Are you concisely changing the tag here? It might result in a broken link.

Reply:
good catch, that is remnant of some testing, no intention to change any tags, will fix it